STYLE.md

```diff
@@ -131,15 +131,188 @@ if status is not pebble.ServiceStatus.ACTIVE:
 
 ## Docs and docstrings
 
-### Use British English
+### Tones
+
+#### Avoid negative, be positive
+
+State conditions positively as what should happen, rather than what shouldn't. Focus on desired behaviour, and frame instructions and conditions around the expected or successful outcome, not failure cases.
```

Comment:
```suggestion
State conditions positively: what should happen, rather than what shouldn't. Focus on desired behaviour, and frame instructions and conditions around the expected or successful outcome, not failure cases.
```